docs/source/examples/workspace/topologies/gce-eip.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Can we start to move away from using the workspace directory in favor of workspaces? The sooner we can remove our dependence on it, the sooner we can delete the dead code